admin/src/scenes/phase0/components/CniField.jsx

```diff
@@ -1,23 +1,13 @@
 import UploadedFileIcon from "@/assets/icons/UploadedFileIcon";
-import { AddButton, DeleteButton, MiniTitle, MoreButton } from "./commons";
 import React, { useEffect, useState } from "react";
-import api from "@/services/api";
-import { translate, download } from "snu-lib";
-import { toastr } from "react-redux-toastr";
 import CorrectionRequest from "./CorrectionRequest";
 import PencilAlt from "@/assets/icons/PencilAlt";
 import CorrectedRequest from "./CorrectedRequest";
-import { Modal } from "reactstrap";
-import { BorderButton, PlainButton } from "./Buttons";
-import ConfirmationModal from "./ConfirmationModal";
-import Warning from "@/assets/icons/Warning";
-import { capture } from "@/sentry";
-import Field from "./Field";
-import DatePickerInput from "@/components/ui/forms/dateForm/DatePickerInput";
-import { resizeImage } from "../../../services/file.service";
-import Eye from "../../../assets/icons/Eye";
-import { Spinner } from "reactstrap";
-import Download from "../../../assets/icons/Download";
+import { HiDotsVertical, HiExclamation, HiOutlineInformationCircle } from "react-icons/hi";
+import ReactTooltip from "react-tooltip";
+import { isBefore } from "date-fns";
+import classNames from "classnames";
```

Comment:
```suggestion
import cx from "classnames";
```
on a l'habitude de réduire son nom par pas allonger les lignes